front/lib/models.ts

```diff
@@ -1098,7 +1103,10 @@ EventSchema.init(
   {
     modelName: "event_schema",
     sequelize: front_sequelize,
-    indexes: [{ fields: ["workspaceId", "marker"], unique: true }],
+    indexes: [
+      { fields: ["workspaceId", "marker"], unique: true },
+      //{ fields: ["sId"], unique: true }, @todo daph add this after migration 20230829_extract_sids
```

Comment:
I think this could be switched on as is ? unique doesn't count for `NULL` I believe